x-pack/plugins/security_solution/server/lib/entity_analytics/risk_engine/routes/disable.test.ts

```diff
@@ -15,6 +15,29 @@ import {
   requestMock,
 } from '../../../detection_engine/routes/__mocks__';
 import { riskEngineDataClientMock } from '../risk_engine_data_client.mock';
+import { securityMock } from '@kbn/security-plugin/server/mocks';
+
+export const createMockSecurityStartWithFullRiskEngineAccess = () => {
+  const mockSecurityStart = securityMock.createStart();
+
+  const mockCheckPrivileges = jest.fn().mockResolvedValue({
+    hasAllRequested: true,
+    privileges: {
+      elasticsearch: {
+        cluster: ['manage', 'monitor'],
+        index: {
+          'index-name': ['read'],
+        },
+      },
+    },
+  });
+
+  mockSecurityStart.authz.checkPrivilegesDynamicallyWithRequest = jest
```

Comment:
Nit: It would be lovely to make this the default behavior in tests: either the `with` function is defined to just invoke the child (and not do any actual checks), or security is mocked to return the basic permissions we need. We do a lot of that kinda stuff in https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/server/lib/detection_engine/routes/__mocks__/request_context.ts. 

Also: this is a vague idea that may/may not pan out. If you want to explore it, I'd be happy to pair on it! I could use a refresher on that stuff, myself.

Reply:
I will have a look into this tomorrow, if you get to re-reviewing and this is the only outstanding item I would prefer to merge as-is and address in another PR if possible.

Reply:
Yep, that makes sense to me!